sdk/tables/data-tables/test/integration/batch.spec.ts

```diff
@@ -9,117 +9,113 @@ import { isNode } from "../testUtils";
 import { Uuid } from "../../src/utils/uuid";
 import * as sinon from "sinon";
 
-if (isNode || isLiveMode()) {
```

Comment:
This is the only line removed in this file. The rest is formatting from the prettier commit hook